python/test/executor_test.py

```diff
@@ -78,9 +78,11 @@ def dtype_to_mscclpp_dtype(dtype):
 
 
 def main(
-    execution_paln_name: str,
+    execution_plan_name: str,
     execution_plan_path: str,
     size: int,
+    nthreads_per_block: int,
```

Comment:
We don't need this arg, it will be included in execution plan and default value is 1024